airtime_mvc/application/controllers/PreferenceController.php

```diff
@@ -262,6 +262,41 @@ public function streamSettingAction()
                     //Application_Model_RabbitMq::PushSchedule();
                 }
 
+                // pulling this from the 2.5.x branch
+                if (!Application_Model_Preference::GetMasterDjConnectionUrlOverride()) {
+                    $master_connection_url = "http://".$_SERVER['SERVER_NAME'].":".$values["master_source_port"].$values["master_source_mount"];
+                    if (empty($values["master_source_port"]) || empty($values["master_source_port"])) {
```

Comment:
Is it an edge case though? Most folks that will be using the instreamer will do so from the internet and I'm not sure everyone has the means to do this over a proper VPN tunnel. For smaller stations using a consumer grade internet uplink I always saw this as a handy way for them to communicate the proper IP or DynDNS entry to staff that needs it.